src/parser/shared/modules/hit-tracking/utilities.ts

```diff
@@ -0,0 +1,17 @@
+import Enemies from 'parser/shared/modules/Enemies';
+import { DamageEvent } from 'parser/core/Events';
+import MAGIC_SCHOOLS from 'game/MAGIC_SCHOOLS';
+
+import IgnoredAbilities from './IgnoredAbilities';
+
+export function notableEnemy(enemies: Enemies, event: DamageEvent): boolean {
+    return !event.sourceIsFriendly || event.sourceID === undefined || !enemies.getEntities()[event.sourceID];
```

Comment:
logic here seems weird; returns true (is notable) if the sourceID is undefined or the enemy entity doesnt exist?

Reply:
yup, i flipped the logic in it on accident